app/src/main/java/com/github/se/cyrcle/ui/list/ListScreen.kt

```diff
@@ -147,16 +138,35 @@ fun SpotListScreen(
                 HorizontalDivider(
                     thickness = 1.dp, color = MaterialTheme.colorScheme.primary.copy(alpha = 0.2f))
               }
+              // The parkings are sorted by distance to the user's location (default being the EPFL)
```

Comment:
That's really a detail but this comment is the same as the one below at line 186, yet this one is for pinned parkings while the other is for unpinned parkings. Just so we dont get confused, mention in this comment that these parkings are the pinned ones.

Reply:
Technically it's not even a problem since a bit above it's mentioned if it's for all parkings or pinned parkings

Reply:
I think it is clear because like you said it is mentionned just a few lines above but tell me if it bothers you !

Reply:
I will add pinned then.

Reply:
done !